calicoctl/commands/resources.go

```diff
@@ -101,31 +101,32 @@ func getResourceFromArguments(args map[string]interface{}) (unversioned.Resource
 	orchestrator := argutils.ArgStringOrBlank(args, "--orchestrator")
 	resScope := argutils.ArgStringOrBlank(args, "--scope")
 	switch strings.ToLower(kind) {
-	case "node", "nodes":
+	case "node", "nodes", "no", "nos":
 		p := api.NewNode()
 		p.Metadata.Name = name
 		return *p, nil
-	case "hostendpoint", "hostendpoints":
+	case "hostendpoint", "hostendpoints","hostep", "hosteps", "hep", "heps":
 		h := api.NewHostEndpoint()
 		h.Metadata.Name = name
 		h.Metadata.Node = node
 		return *h, nil
-	case "workloadendpoint", "workloadendpoints":
+	case "workloadendpoint", "workloadendpoints", "workloadep", "workloadeps",
+		"wlep", "wleps", "wep", "weps":
 		h := api.NewWorkloadEndpoint()
 		h.Metadata.Name = name
 		h.Metadata.Orchestrator = orchestrator
 		h.Metadata.Workload = workload
 		h.Metadata.Node = node
 		return *h, nil
-	case "profile", "profiles":
+	case "profile", "profiles", "pro", "pros":
 		p := api.NewProfile()
 		p.Metadata.Name = name
 		return *p, nil
-	case "policy", "policies":
+	case "policy", "policies", "pol", "pols":
 		p := api.NewPolicy()
 		p.Metadata.Name = name
 		return *p, nil
-	case "ippool", "ippools":
+	case "ippool", "ippools", "ipp", "ipps", "ip", "ips":
```

Comment:
I think `ip` and `ips` aren't good choices here given that an IP address is a real thing that might exist in our data model.

Maybe `pool` though?